src/utils/components/withLoading.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import withLoading from './withLoading';
+
+const dangerousHTML = '<div>Dangerously setting inner HTML</div>';
+const makeDanger = () => ({__html: dangerousHTML});
+
+const testComponentClass = 'testClass';
+const TestComponent = (props) => (<div {...props} />);
+const TestComponentWithLoading = withLoading(TestComponent);
+
+describe('withLoading', function() {
+	const testComponentWithLoading = shallow(<TestComponentWithLoading className={testComponentClass} />);
+
+	it('matches snapshot', () => {
+		expect(testComponentWithLoading).toMatchSnapshot();
+	});
+
+	it('renders a wrapped component', () => {
+		expect(testComponentWithLoading.find(`.${testComponentClass}`).length).toBe(1);
+	});
+
+	it('can handle dangerouslySetInnerHTML', function() {
+		const dangerousHTMLComponent = shallow(
+			<TestComponentWithLoading
+				dangerouslySetInnerHTML={makeDanger()}
+			/>
+		);
+		expect(dangerousHTMLComponent.html()).toContain(dangerousHTML);
+	});
```

Comment:
I'd like two more tests with children passed into `TestComponentWithLoading`, e.g.

```
testComponentWithLoading = shallow(<TestComponentWithLoading><h1>Hi</h1></TestComponentWithLoading>);

...

testComponentWithLoading = shallow(
  <TestComponentWithLoading>
    <h1>Hello</h1>
    <p>World</p>
  </TestComponentWithLoading>
);
```

Since the HOC does some special manipulation of children, you should check that the children are being rendered the way you expect. These should also probably be snapshots to make it super-explicit how things are rendered